src/Libraries/Microsoft.Extensions.AI/ChatCompletion/ChatClientStructuredOutputExtensions.cs

```diff
@@ -149,7 +147,7 @@ public static async Task<ChatResponse<T>> GetResponseAsync<T>(
         IEnumerable<ChatMessage> messages,
         JsonSerializerOptions serializerOptions,
         ChatOptions? options = null,
-        bool? useJsonSchema = null,
+        bool? useJsonSchemaResponseFormat = null,
```

Comment:
Couldn't we just make this a regular boolean in all methods?
```suggestion
        bool useJsonSchemaResponseFormat = true,
```

Reply:
I believe Steve 's intent was we might be able to detect in the future whether the underlying provider supports structured output or not